libs/libqmi/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libqmi
+PKG_VERSION:=1.22.4
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://anongit.freedesktop.org/libqmi
```

Comment:
Official tarballs are preferred. This way we can get rid of autoreconf as well as having a simpler Makefile.

Reply:
Agree

Reply:
Will do